bin/powershell_helper.rb

```diff
@@ -0,0 +1,36 @@
+#! /usr/bin/env ruby
+#
+#   powershell_helper.rb
+#
+# DESCRIPTION:
+#   This is meant to execute powershell in the bin/powershell deirectory
+#   so we don't have to copy the code out of the GEM. This is meant only as a
+#   wrapper.
+#
+# OUTPUT:
+#   N/A
+#
+# PLATFORMS:
+#   Windows
+#
+# DEPENDENCIES:
+#   None
+#
+# USAGE:
+#   C:\opt\sensu\embedded\bin\ruby C:\opt\sensu\embedded\bin\powershell_helper <poweshell-script> ARGS
+#
+# NOTES:
+#
+# LICENSE:
+#   Copyright 2015 <dasecretzofwar@gmail.com>
+#   Released under the same terms as Sensu (the MIT license); see LICENSE for details.
+#
+
+if ARGV.empty?
```

Comment:
rather than using argv you could depend on the `sensu-plugin` gem library which gives a better interface in my opinion as you can have descriptions, option names, etc.